11ty/definitions/insane.md

```diff
@@ -1,8 +1,33 @@
 ---
 title: Insane
 slug: insane
-defined: false
+defined: true
+speech: adj
+alt_words:
+  - bizarre
+  - intense
+  - odd
+  - perculiar
+  - weird
 flag:
  level: avoid
  text: ableist language
----
\ No newline at end of file
+reading:
+  - text: Words of the Week – “Insane and Insanity”
+    href: https://therollingexplorer.com/ableist-language-to-avoid-and-acceptable-alternatives-insane-edition/
+---
+
```

Comment:
To maintain our structure, we should start this definition including its various usages:

>1.  in a state of mind which prevents normal perception, behavior, or social interaction; seriously mentally ill.
>2. (of an action or quality) characterized or caused by madness.
>3. colloquially, used to mean to an intense degree, such as _extreme_, _extraordinary_, _shocking_, _outrageous_